### PR TITLE
Chore/4249/brand audit

### LINKED ISSUE #4165: Design System - Add containers
https://www.figma.com/file/2MLHeIeL6XPVi3Tc2DfFCr/%E2%9D%96-Leather-%E2%80%93-Design-System?type=design&node-id=5056-60851&mode=design&t=CbvgKq8c4jj6eRZE-0

Screenshot 2023-10-03 at 14 28 01

Screenshot 2023-10-03 at 14 27 33

### LINKED ISSUE #4249: chore: brand audit changes
We went through the brand audit today and there are some things to fix. We decided to fix some small things soon but tackle the others when re-working the UI foundation. 

- Where spacing / layout is concerned we will tackle those in https://github.com/leather-wallet/extension/issues/4165
- Add icons to mnemonic page buttons ( will do in https://github.com/leather-wallet/extension/issues/4130 )

Quick fixes for now:
- Change font of collectibles header to `label 1`
- Change tab hover BG color to `component-bg-hover` 
- Set minimum height of main container in Assets / activity to be the same based on the height of our empty assets screen
- Change text colour of BTC address secondary to subdued
- Fix colors in Send screen ` empty 0 and Enter recipient address`

Brand Audit:
https://www.figma.com/file/GU3X88E6UJWA3JtbP7FThe/%F0%9F%94%AC-Brand-v0-Audit?type=design&node-id=1%3A125&mode=design&t=h0oX07Q0cFocF07j-1 

CC @mica000 @markmhendrickson @fabric-8

### LINKED ISSUE #4290: Fix refresh icon alignment for collectibles
Image

### PR DESCRIPTION
**Pull request by pete-watters**
> Try out this version of Leather — [download extension builds](https://github.com/leather-wallet/extension/actions/runs/6507767401).

This PR fixes some issues flagged in the:
- Brand audit ticket #4249 
    - [fix: update address displayer to use text-subdued for odd](https://github.com/leather-wallet/extension/pull/4343/commits/02d035b4ab94d6442cbbcc69833c367f8d3e8a1f)
    - [fix: change tab hover bg colour,](https://github.com/leather-wallet/extension/pull/4343/commits/3b4bdc47002138600079f43f36a4e51b728b5bdf)
    - [fix: give activity screen a min height to stop it jumping when empty, c…](https://github.com/leather-wallet/extension/pull/4343/commits/9036f3b1cfaac1cade8f5f14227eb0148734f1de)
- Containers ticket #4165
    - [fix: alter position of receive modal in extension view,](https://github.com/leather-wallet/extension/pull/4343/commits/0243608fbf4659dc32808d38a27625b8cca40d74)
    - [fix: stop create account button disappearing at smaller heights,](https://github.com/leather-wallet/extension/pull/4343/commits/bce857c9110388ae21cb3f6b7ee84f75db4b0baf)
    - [fix: improve display of select account in extension mode,](https://github.com/leather-wallet/extension/pull/4343/commits/1a0b49ac2407a25db031bb5a34f6c8c301b168e2)
- Align refresh icon #4290 
    - [fix: update collectible header font and align icon,](https://github.com/leather-wallet/extension/pull/4343/commits/ea4fd8ade1dc2d13e3f5e6bd687ac3d9bbadeb1d)

### CODE REVIEW

File: src/app/components/drawer/components/drawer-header.tsx [hunk 1]
```diff
@@ -29,22 +28,15 @@ export function DrawerHeader({
...
   return (
-    <Flex
-      pb="base"
-      justifyContent="space-between"
-      alignItems="center"
-      pt="loose"
-      px="loose"
-      {...bind}
-    >
+    <Flex justifyContent="space-between" alignItems="center" py="space.04" px="space.04" {...bind}>
```
Comment (fbwoolf) [inline src/app/components/drawer/components/drawer-header.tsx]: It looks like it was `pt` here, but now `py`? If that is the case, can it just be `p="space.04"` bc same?

File: src/app/components/drawer/controlled-drawer.tsx [hunk 2]
```diff
@@ -17,6 +17,9 @@ export function ControlledDrawer(props: ControlledDrawerProps) {
 
   return (
     <BaseDrawer
+      height="100vh"
+      position={['absolute', 'fixed']}
+      top={['-24px', 0]}
```
Comment (fbwoolf) [inline src/app/components/drawer/controlled-drawer.tsx:22]: Is there a screenshot of what you are fixing here? I'm a bit lost on why I'm not seeing anything broken on my branch with the `BaseDrawer`. 🤔 Maybe I'm just not understanding the change here?

File: src/app/components/text-input-field.tsx [hunk 3]
```diff
@@ -119,6 +119,7 @@ export function TextInputField({
           ref={inputRef || ref}
           _disabled={{ bg: color('bg') }}
```
Comment (fbwoolf) [inline src/app/components/text-input-field.tsx]: Shouldn't `color` from @stacks/ui be replaced here?

File: src/app/features/switch-account-drawer/components/create-account-action.tsx [hunk 4]
```diff
@@ -7,8 +7,16 @@ interface CreateAccountActionProps {
...
+    <Flex
+      mt="100px"
+      py="space.05"
+      px="space.05"
+      flexGrow="1"
+      position="fixed"
+      bottom={0}
+      width="100%"
+    >
+      <LeatherButton data-testId="create-account" fullWidth onClick={() => onCreateAccount()}>
```
Comment (fbwoolf) [inline src/app/features/switch-account-drawer/components/create-account-action.tsx]: Shouldn't this be `data-testid`?

Comment (pete-watters) [reply src/app/features/switch-account-drawer/components/create-account-action.tsx]: Thanks, I'll remove this as it's not used in tests, it was just helping me search the DOM

File: src/app/components/drawer/components/drawer-header.tsx (hunk 1 above)
Comment (pete-watters) [reply src/app/components/drawer/components/drawer-header.tsx]: Thanks, I'll update

File: src/app/components/drawer/controlled-drawer.tsx (hunk 2 above)
Comment (pete-watters) [reply src/app/components/drawer/controlled-drawer.tsx:22]: I put some screenshots and more explanation in the ticket linked with this commit: 
https://github.com/leather-wallet/extension/issues/4165#issuecomment-1757781322

I'm doing this to close the gap between the top of the modal and the top of the extension: 

https://github.com/leather-wallet/extension/assets/2938440/b0fe260d-d347-4494-83ef-5a38b2a08df4

If you see  that comment, it seems like most of the modals we use have this gap in extension mode. 

https://github.com/leather-wallet/extension/assets/2938440/f9143fcc-1e6f-4516-8444-85bd61d32179

I have this task to add containers - https://github.com/leather-wallet/extension/issues/4165 but on review of that I'm not sure if that will solve this. 

We have issues in extension mode with how we display our modals. If that containers task is to fix the issue we will be changing most modals into full page views which I'm not sure we want to do. 

CC @kyranjamie - maybe we need to catch up on this?

File: src/app/components/text-input-field.tsx (hunk 3 above)
Comment (pete-watters) [reply src/app/components/text-input-field.tsx]: It should, thanks! I'll actually remove this change as it's not fully working, thanks!

I had opened a new ticket to apply the updated placeholder colours so I can include changing the disabled state colour also. 

https://github.com/leather-wallet/extension/issues/4345

File: src/app/features/activity-list/components/tab-wrapper.tsx [hunk 5]
```diff
@@ -0,0 +1,15 @@
+import { Box } from 'leather-styles/jsx';
+
+interface TabWrapperProps {
+  children: React.ReactNode;
+  padContent?: boolean;
+}
+
+export function TabWrapper({ children, padContent = false }: TabWrapperProps) {
```
Comment (kyranjamie) [inline src/app/features/activity-list/components/tab-wrapper.tsx]: Wonder if this name should be more specific to activity list? Will we use it elsewhere (presume not as in this folder)

**Review by kyranjamie — APPROVED** [review]

File: src/app/features/activity-list/components/tab-wrapper.tsx (hunk 5 above)
Comment (pete-watters) [reply src/app/features/activity-list/components/tab-wrapper.tsx]: Yes, good idea. I've updated it. 

More generally we should add our own `Tab` wrapper to the UI library and make sure all tabs always have a set minimum height regardless of content

**Comment by mica000** [comment]
@pete-watters shall we revert the hard angles to the rounded corners and deal with this is separated ticket?  Screenshot 2023-10-16 at 13 03 09

**Comment by pete-watters** [comment]
> @pete-watters shall we revert the hard angles to the rounded corners and deal with this is separated ticket? Screenshot 2023-10-16 at 13 03 09

I didn't change those. Are we going to remove all rounded corners? I can just do that first if you want and get it merged. Then we don't have to add them and remove them again

**Pull request merged**